dataprofiler/profilers/profile_builder.py

```diff
@@ -639,12 +639,6 @@ def update_profile(self, data, sample_size=None, min_true_samples=None):
                           "profiled.")
             return
 
-        # set file properties since data will be processed
```

Comment:
moved in case `_update_profile_from_chunk` says the data is not profile-able with the given profiler e.g. unstructured + multicol dataframe. Hence, doesn't save the file being processed when an error is given.